src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
```

Comment:
Don't set this `text-transform` for the whole page; you should set it for a particular element.

Reply:
done